.github/workflows/release.yml

```
@@ -0,0 +1,56 @@
+name: Build Release
+
+on:
+  release:
+    types: [ published ]
+  workflow_dispatch:
+    inputs:
+      ref:
+        description: ref to build binary from
+        required: false
+  push:
+    branches:
+      - 'frank/build-release'
+
+jobs:
+  build-binary:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        cpu: [ "", "skylake" ]
```

```suggestion
        cpu: [ ""]
```

Per the failing CI, probably easiest to remove "skylake" and just build for regular CPU.

---

Nah, its something that v4 of the upload artifacts breaks, so downgrading.

https://github.com/actions/upload-artifact/issues/478